node.gyp

```diff
@@ -151,8 +152,15 @@
       'type': '<(node_target_type)',
 
       'dependencies': [
-        'node_js2c#host',
-        'deps/nghttp2/nghttp2.gyp:nghttp2'
+        'node_js2c#host'
+      ],
+
+      'conditions': [
```

Comment:
Only `config.gypi` is runtime generated, `node.gypi` is human generated. AFAICT this is the relevant section https://github.com/nodejs/node/blob/ffed7b6e6699deeca18b15bd5f59b4bc97f612e7/node.gypi#L246-L260